tests/system/action/projector/test_next.py

```diff
@@ -0,0 +1,53 @@
+from tests.system.action.base import BaseActionTestCase
+
+
+class ProjectorNext(BaseActionTestCase):
+    def setUp(self) -> None:
+        super().setUp()
+        self.set_models(
+            {
+                "meeting/1": {},
+                "projector/2": {"meeting_id": 1},
+                "projector/3": {
+                    "current_projection_ids": [1, 2],
+                    "preview_projection_ids": [3, 4],
+                    "history_projection_ids": [],
```

Comment:
Add an element here, so you can check the weight handling.

Reply:
Add weight to the projection/1 to check the weight.

Reply:
Also add a new projection to the history with some weight and check, that the order after calling `next` is right.

Reply:
Add a new projection to history of projector 3.  Add weight to the projections.